neo4j/bookmarks.go

```diff
@@ -61,24 +64,24 @@ type BookmarkManagerConfig struct {
 	// Hook called whenever bookmarks for a given database get updated
 	// The hook is called with the database and the new bookmarks
 	// Note: the order of the supplied bookmark slice is not guaranteed
-	BookmarkUpdateNotifier func(string, Bookmarks)
+	BookmarkUpdateNotifier func(context.Context, string, Bookmarks)
```

Comment:
Note (Unrelated to this PR): most drivers + the ADR call this thing consumer instead of notifier. I guess because it gets notified and doesn't notify itself.

Reply:
I realize that Notifier does not necessarily means what I think it means.
Let me rename that to consumer.

Reply:
fixed in 5fd8c57f9bf4f513d84df1f59b472585a2b3b9a2